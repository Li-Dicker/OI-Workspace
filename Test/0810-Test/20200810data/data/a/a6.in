10
199
43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 43663 
19582 48465 26270 6901 3671 41741 45909 34462 45651 42860 24873 47028 49397 31602 20277 23419 19885 21187 35227 7173 26714 21527 6096 28335 36202 37115 3206 35908 14881 46786 40247 45784 48139 19252 5298 5008 19613 42427 40208 35875 35351 6662 48240 30864 25895 525 17199 37806 14677 43698 24613 4096 47618 28026 46832 42588 6353 25144 37603 41986 3749 41015 27107 38249 41913 33169 32024 49157 11517 7028 35233 26378 5611 7255 32694 26572 20782 32003 1482 7974 25050 29779 27061 41469 47593 6791 21914 29642 15713 3877 2197 6417 25517 23758 30817 9865 26130 13529 46140 27825 10102 26746 27541 10640 23390 37965 14426 45711 33042 7270 14533 32024 28622 45839 47075 46046 32891 15255 5181 23146 21213 17289 16696 16623 23631 22463 1409 8059 35618 8783 34195 32583 20898 2106 29250 2657 12129 45160 4535 45050 30286 26517 41550 26730 26453 3442 29833 14912 2136 48459 39378 42606 4926 14190 5591 31768 29148 47318 29596 44071 9716 1547 38999 19141 49460 3683 7265 15029 10833 47000 29911 48565 2198 3747 41170 38494 35442 43833 40890 21706 36746 12413 10757 18551 36202 44790 19936 20340 12097 10233 21594 37043 48787 41690 34685 11837 26482 24399 44004 
199
16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 16622 
41613 31745 32028 23212 48305 5072 6049 37268 16450 39079 31792 27784 33765 14914 43571 44063 29170 7086 44087 49931 43350 47830 16752 1186 5220 2532 33768 46536 26921 7459 36434 36051 3361 42151 28335 7265 24537 5748 40867 14350 23259 46564 45773 23167 3705 17095 23335 12860 6798 46692 22221 3285 7294 43827 8347 30433 17116 18690 27121 27674 33180 31071 48188 1115 10315 35642 26997 14008 45996 13633 30181 22425 36447 15422 19583 18625 23681 32511 46986 40521 272 4599 34581 19230 38493 11193 48720 28145 37085 24935 49412 31512 39439 25576 35345 9077 6511 13690 8625 5646 7794 45690 32539 9543 41808 6114 47311 4234 29143 38534 17895 40267 43330 1958 46939 42015 48620 17641 47114 15116 34387 12352 17470 34710 39777 14507 47489 720 49853 27790 41111 3991 16398 28257 43770 11310 23417 27378 38882 41867 10383 4909 19110 28414 27615 14376 15466 19314 19671 48088 6567 30769 29956 4541 12272 35998 22066 27657 18364 30638 46796 44947 22299 35366 40387 29719 33420 38196 49838 28951 2501 38830 49736 5892 8820 24699 36865 28535 26288 31264 27439 41962 48743 8313 25821 48823 649 32384 32083 41677 41322 22382 39705 21468 1428 2651 40713 46714 47317 
199
79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 79960 
35521 46848 9059 37941 8613 7696 46491 39191 47908 40158 44170 17042 48745 25460 1069 6508 31146 24048 49355 8425 40953 5332 23354 4300 40846 2173 4456 31548 11088 30613 17364 5596 22367 7798 35267 5122 46478 12781 4794 46504 33644 13069 5283 48129 34565 36035 49543 41331 39200 39371 29469 12262 24965 16565 11834 27738 32367 3897 40836 42910 40158 17724 26419 48868 2712 39316 1144 6125 48259 44906 37542 15728 12036 20594 17765 16836 47458 12360 17784 11321 10859 16819 35391 11994 26093 27074 17432 49628 1141 22845 4377 20183 15689 12155 37575 12831 27431 718 45866 14433 23086 18556 12514 49261 48827 26201 43868 18364 25869 48236 21822 46494 8691 46063 16624 28686 12096 21024 24494 29297 23239 47 30083 48216 3410 14915 16188 15780 40856 32371 16514 33485 16001 15478 36803 47825 42829 23035 30338 25833 335 37489 30828 26886 1193 13676 3714 43057 17663 40156 25270 40215 45051 46580 48162 13571 33137 6413 13476 26032 1461 49827 16381 22650 34435 6391 40937 48005 8363 43497 37735 38926 27957 41153 46713 32513 2352 17257 22698 40288 13671 10703 16522 4127 6762 48654 1545 49399 23711 31147 5513 5646 8398 10683 36363 30552 20312 15108 27190 
196
33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 33897 
22367 45351 49916 46994 11253 40198 37933 40652 11181 47005 2187 21788 28218 43361 3245 18222 43746 1993 8754 2129 32197 3725 10962 5770 24097 37325 9637 25523 15190 48711 952 18294 19409 41877 15923 42415 45708 32147 22408 9278 29952 6811 39406 39329 6828 37341 3868 18936 4290 12440 38809 18128 29190 48171 5611 47018 21720 1354 30439 1912 47977 36620 36886 24464 5710 3205 36162 15704 29393 21405 49062 34846 44205 38576 47177 40925 21750 44231 24193 20205 20349 5082 9515 36778 15799 32146 29548 24493 27643 46786 39243 44728 7139 6178 21710 5644 21021 12064 22116 34720 48953 9285 27638 31800 49564 21546 19390 12723 5869 10499 37770 11277 16832 44694 40990 20449 44899 17291 4265 550 43001 11500 15152 9793 15541 28018 9322 36245 45814 24902 46296 1807 32791 19175 38259 19705 21325 41513 4777 47412 46861 24707 16427 46912 23213 49470 47137 23885 34869 44923 24696 20287 28251 20932 18595 26326 44320 8626 528 17237 29138 12029 14000 28643 34782 43086 14134 39542 31646 27725 346 6300 32421 23689 49250 3940 3682 4532 7368 23572 46517 7750 32681 38126 17297 20894 2090 34620 28754 10704 46111 38567 35933 48547 27389 14456 
200
42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 42620 
15943 20568 18353 30568 26439 6416 33783 27932 5338 44885 11456 9485 33983 38186 1234 35041 37368 34119 20320 33412 39358 24046 8046 47074 33061 10551 27356 26224 49164 28498 3390 9261 788 8847 33651 23454 49417 22178 44129 23991 15924 33907 3895 32981 698 11198 11350 43599 40508 39931 33202 20549 40638 38487 47647 1741 10466 16166 21631 45297 46242 47257 31588 47171 49087 6670 44182 20438 27073 1001 44589 29251 40456 42667 34891 26694 29658 15897 8185 10074 17715 43929 35 37145 49635 40146 42644 7861 41039 10490 2839 46514 15488 47374 20044 4529 23798 36172 28047 26721 4097 6348 21574 46478 36692 13258 36695 25144 46559 37126 46418 46200 41950 29940 19970 6659 28964 36948 41145 23709 14636 1850 8746 21604 25369 31623 10348 28283 12696 32152 20977 28878 36201 11006 11472 16045 35887 47104 11935 14567 44031 6637 15213 30962 309 748 15596 16365 49150 43077 14746 2774 6973 12994 39558 37152 33653 24852 23712 15348 45974 49511 7308 44858 32184 28438 24159 43836 16151 22459 14911 7712 7073 37557 24064 46442 17454 4295 40599 47327 18323 43042 28790 6451 42372 44645 48902 20769 44689 183 6209 45386 47289 43402 16538 32955 25712 14197 10138 4358 
198
20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 20047 
22651 4669 31036 46937 9519 14764 4248 3434 25176 33827 46761 2189 29975 8781 32878 42572 23337 39563 40524 18173 5100 24555 26542 10071 26779 39748 35924 46026 25881 19280 18218 5162 29898 17784 36363 6141 17292 16030 22805 23263 19673 5030 9269 40059 39666 24104 35374 28504 1856 44259 2567 46849 6668 27308 30549 49458 15901 45828 43024 46336 12118 38438 15519 29574 28660 16669 5881 42828 17981 34776 23189 41279 16497 28932 21036 34131 48138 17107 7027 10479 13580 36503 42720 17347 47344 3096 14477 11364 14558 34294 14701 46522 2613 47027 39358 36460 6767 49703 26414 49156 33257 8339 33996 47903 13889 19084 12337 19694 36333 15468 18470 1698 17384 45019 4731 480 10820 13424 48598 41444 16871 45979 8462 20150 32121 1372 28293 5580 28330 40123 23756 1448 46144 9916 35821 16020 19363 6294 24144 15105 30401 19975 5841 38852 121 3373 21610 15664 33610 33883 37339 19793 37566 10969 28722 6626 16769 34361 27095 38438 41899 5239 5080 19818 7964 7916 1024 7822 12506 18232 46035 24305 42352 39374 3887 27533 31392 48225 37108 30592 38409 30461 38555 43764 39693 31098 31289 9168 22381 47780 5670 44442 14977 47941 28951 49520 14159 22419 
196
5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 5098 
39861 641 8023 37055 33005 45834 20394 22909 20880 9734 21217 39496 20359 35412 30219 9555 15147 42023 8077 29686 7135 49323 48545 3151 18272 3575 4746 14428 16685 8664 40489 46092 3390 41932 49050 47869 26409 44293 6923 9801 45645 40932 20456 18768 37462 42020 27980 18274 6331 24881 25982 6325 27460 2920 49943 33163 26312 42666 44962 23634 40869 36794 10047 29016 25367 18802 14621 10805 43130 7031 11013 23222 23890 14130 20369 34940 28229 31910 28559 17440 39453 12701 48273 40854 33872 1227 16997 28590 15770 45216 41208 1046 471 44877 20202 8884 6850 13168 40617 10528 442 32157 34756 3025 39387 30932 45450 292 39251 35623 8548 30711 32083 9865 41175 27983 31926 49611 25968 17567 3881 21456 26607 31693 44480 313 4429 4146 33392 45923 34658 45514 21830 8498 8174 45279 27021 2872 45053 49128 7792 35419 44517 21046 8049 41359 22638 37515 27153 35851 42038 43740 48277 19502 32559 16107 7735 15712 32391 23644 23283 24147 38808 20776 43501 37282 8198 33604 39214 31552 10902 35797 6976 36672 44406 46570 23926 13262 36840 23463 17504 15314 35230 36942 39656 48165 9419 6547 26808 11310 18960 35828 18258 48384 46572 25490 
198
29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 29912 
32695 40315 38080 36405 16814 35754 40048 39479 2779 26359 3082 35514 43626 6887 13707 19731 31207 22528 24289 35629 6132 16360 28787 3151 21398 32675 41278 18888 37762 35758 11963 15252 40905 26377 32967 46928 13002 31609 23699 19678 47367 48560 46928 16774 23929 13118 31048 20613 4229 43337 24162 43235 6126 16200 10758 7951 25029 33984 46793 15066 10342 4886 12678 733 3263 20804 13120 36430 39413 25151 32885 38550 2483 11631 16654 40626 46553 6597 46116 39459 21105 10957 48849 24265 40876 747 46082 42767 25454 45945 22232 43447 23222 38638 16850 29644 6749 37712 2787 26207 20286 3897 46346 3254 34117 15495 4800 21815 33676 44831 37292 11022 19508 48416 45342 3828 47735 44103 3476 4408 48989 37826 29761 31894 41674 3148 47375 23318 34918 40105 24493 3112 1604 7089 42801 22351 33394 25629 27347 23371 10572 47768 48468 16967 16786 47365 2725 5637 28199 318 31010 5247 31274 21268 11220 22413 44786 36296 38119 18930 34097 15516 48000 3975 29326 197 18225 40255 29050 6070 28529 37995 37003 27730 11142 9279 31685 29130 39711 7919 21320 3124 23934 40727 45507 7937 44312 42869 43018 38683 4045 36398 25425 33183 25481 31464 37017 24162 
198
33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 33190 
21663 11661 37228 1267 536 10810 34780 8169 17464 34380 6618 3215 17627 18161 19943 9705 47971 8732 1436 8234 37172 317 29406 37463 29232 28021 35883 35569 10971 16760 35679 5021 7369 45826 30498 30989 40876 8952 14181 37220 1422 14648 3823 21247 46850 39031 49061 39250 11199 28272 26626 27832 7535 3832 7995 17539 40052 39398 9398 16658 4801 4336 13606 22315 11801 28592 43784 30141 1991 48637 16228 22641 32789 40850 5338 38865 16369 18925 1814 19986 38405 43495 3493 7897 39440 45628 47428 30538 12840 39655 22657 32538 22469 18376 38211 5287 17194 33436 8674 19411 47845 8835 29398 6151 29299 37493 28578 45073 47416 22036 42883 42969 34289 33388 23623 35348 11709 39506 8925 14060 42611 29162 38957 26305 11063 37884 17264 23968 16529 44858 30198 49164 48657 7166 29727 32283 828 1159 45140 46080 20871 14229 22799 38554 5226 15916 39956 8457 9378 41697 291 2612 33706 31732 40222 31450 17699 29751 6812 26647 37228 45635 4060 11853 1154 20906 5691 5170 33766 28407 17624 40192 33129 23810 46939 21263 40385 46602 48892 12446 34686 1888 34445 7263 48373 10482 24215 43857 8397 35399 9811 13005 23970 18569 7338 7843 36560 36604 
200
45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 45245 
5705 11898 35099 24026 2302 4162 46559 41571 19832 11469 7793 28946 29491 46809 12530 30977 15197 3554 25804 47520 10036 1253 40691 35676 11279 29212 47517 35597 15780 49222 13335 14092 30886 2255 31161 36271 14292 20431 514 40202 46328 43102 18285 21288 34675 33234 8000 22127 16993 30085 27518 17574 4204 14579 20718 45622 25932 24007 43177 4868 46527 33586 44479 2998 20503 39467 46009 4947 4782 34480 47708 13477 2404 21439 28332 41733 6814 1239 25229 22518 44574 16279 37802 39482 22376 31277 19644 10786 6333 4529 10928 25573 28664 40964 33746 12747 43777 14323 15085 40887 39581 38479 40520 12884 13154 14865 48180 30397 20656 17517 11421 43994 22672 6741 33497 28165 20335 30375 25012 20936 23459 38490 25550 34041 20735 35629 7270 10696 15137 42477 33160 8287 4820 16688 12653 8537 33017 16950 23655 41540 23020 9736 23388 3191 8483 13284 44534 37520 17678 26483 3991 10997 17238 48389 21429 16575 10626 16296 49143 25228 19338 28835 18444 38328 28726 8162 43026 23640 7433 42962 42959 9375 2491 5589 45916 7470 32971 21212 47753 30772 2957 3167 41007 33154 41414 3306 6112 19135 31714 48937 10660 3348 45645 15582 32939 13859 35012 11278 21572 36241 
