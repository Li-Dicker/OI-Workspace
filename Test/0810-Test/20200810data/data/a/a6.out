Case #1: 13237214
Case #2: 7788886
Case #3: 19940611
Case #4: 10827521
Case #5: 13094981
Case #6: 8114756
Case #7: 5439575
Case #8: 10229591
Case #9: 10523206
Case #10: 13187107
